src/app/dev/DevToys.Tools/Tools/Converters/Date/DateConverterCommandLineTool.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System.Globalization;
+using DevToys.Tools.Helpers;
+using DevToys.Tools.Models;
+using Microsoft.Extensions.Logging;
+
+namespace DevToys.Tools.Tools.Converters.Date;
+
+[Export(typeof(ICommandLineTool))]
+[Name("DateConverter")]
+[CommandName(
+    Name = "dateConverter",
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Converters.Date.DateConverter",
+    DescriptionResourceName = nameof(DateConverter.Description))]
+internal sealed class DateConverterCommandLineTool : ICommandLineTool
+{
+    [CommandLineOption(
+        Name = "number",
+        Alias = "n",
+        DescriptionResourceName = nameof(DateConverter.NumberOptionDescription))]
+    internal long? NumberOption { get; set; }
+
+    [CommandLineOption(
+        Name = "date",
+        Alias = "d",
+        DescriptionResourceName = nameof(DateConverter.DateOptionDescription))]
+    internal string? DateOption { get; set; }
+
+    [CommandLineOption(
+        Name = "epoch",
+        Alias = "e",
+        DescriptionResourceName = nameof(DateConverter.EpochOptionDescription))]
+
+    internal string? EpochOption { get; set; }
+
+    [CommandLineOption(
+        Name = "timezone",
+        Alias = "tz",
+        IsRequired = true,
+        DescriptionResourceName = nameof(DateConverter.TimezoneOptionDescription))]
+    internal string? TimeZoneOption { get; set; }
+
+    [CommandLineOption(
+        Name = "format",
+        Alias = "f",
+        IsRequired = true,
```

Comment:
Since this option has a default, not need to make it required.
![image](https://github.com/veler/DevToys/assets/3747805/ea507a61-f2bf-4b3d-b8f9-7c0d593c94c6)